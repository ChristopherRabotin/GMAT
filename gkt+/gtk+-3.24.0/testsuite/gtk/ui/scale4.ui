<!-- test invalid scale mark attributes -->
<interface>
  <object class="GtkScale">
    <marks>
      <mark value="0" position="between"/>
    </marks>
  </object>
</interface>
