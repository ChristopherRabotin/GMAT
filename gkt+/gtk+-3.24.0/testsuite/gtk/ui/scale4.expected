ERROR: gtk-builder-error-quark 6
.:5:43 Could not parse enum: 'between'
